### PR TITLE
update html nav links and created modules with import and export methods

### PR DESCRIPTION
**Pull request by Besufekad-HAZ**
-Use ES6 methods.
-Create modules to store all js files to import and work with.
-Import the modules to the index.js file.
-Display the current date by importing the module from Luxon.
-Add a live version to the page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,215 @@
...
+- **[Javascript]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- The NAMM SHOW EAST AFRICA WEBSITE SCREENSHOT -->
+
+## 🚀 Screenshot <a name="screenshot"></a>
+
+![image](https://user-images.githubusercontent.com/23375153/217803810-3a3d8691-267c-4303-a68a-76139ddc6d2f.png)
+
```
Comment (levy002) [inline README.md:87]: - [OPTIONAL]  Kindly make sure to add the proper screenshot of this website. The one you have is for another project. Kindly fix that.

**Review by levy002 — APPROVED** [review]
## Hello @Besufekad-HAZ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Professional READEM :heavy_check_mark: 
- Professional PR :heavy_check_mark: 
- Good UI :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Besufekad-HAZ) [reply README.md:87]: Oh...ok my bad... I'll surely fix that...thanks for the feedback. 👍

**Comment by Besufekad-HAZ** [comment]
Finished and Merged the branch

**Pull request merged**